Oxide.Core/OxideMod.cs

```diff
@@ -560,10 +560,20 @@ public void OnFrame(float delta)
             }
         }
 
+        /// <summary>
+        /// Called when the server saves
```

Comment:
This would be a bit misleading, as right now it appears you only have this for Oxide-related, not the server in general.